internal/pass/pairing/memorystore.go

```diff
@@ -0,0 +1,101 @@
+package pairing
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// MemoryStore keeps in memory pairing between extension and mobile.
+//
+// TODO: move cache to separate package.
```

Comment:
Are these comments still relevant? 🤔

Reply:
i will keep todo about ttlcache, right now we are not yet invalidating entries from cache